packages/assets/gulpfile.js

```diff
@@ -0,0 +1,62 @@
+const {
+  src, dest, series, parallel
+} = require('gulp');
+const sass = require('gulp-sass');
+const rename = require('gulp-rename');
+const del = require('del');
+const path = require('path');
+
+sass.compiler = require('sass');
+
+function clean() {
+  return del(['css']);
+}
+
+function compileScss() {
+  return src('./build/*.scss')
+  .pipe(sass({
+    includePaths: [
+      path.join(__dirname, 'scss'),
+      path.join(__dirname, 'scss/00-base'),
```

Comment:
do we need these paths? https://github.com/massgov/mayflower/pull/1240/files#diff-a4d51f0b3500d350fa04a4cfa42579e778eb4d3bdc22eb6f8e575507cd5d61bfR20-R26
I don't see them being used, and just curious, would scss get confused as in build we already have the same file names as the partials in these folders